Controllers/PostController.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Blogger.Data;
+using Blogger.Data.Dto;
+using Blogger.Services;
+
+namespace Blogger.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PostController : ControllerBase
+    {
+
+        private readonly PostService _PostService;
+
+        public PostController(PostService postService)
+        {
+            _PostService = postService;
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> CreatePost([FromBody] PostDto formData)
```

Comment:
formData is not really describing what it is. How about using postRequestData or just post?